From a8fdab22652b2214d73a096d6e0a419d3a16091a Mon Sep 17 00:00:00 2001
From: Manish Regmi <manish.regmi@intel.com>
Date: Thu, 16 Oct 2014 15:58:17 -0700
Subject: [PATCH] Disable clocksource watchdog

Disable the clocksource watchdog. In some cases, this watchdog detects unreliability
in TSC clocksource and tries to switch to different clocksource, but we dont have
any good ones to replace it. so disable it so that it does not try to replace TSC.

Change-Id: Ib8a9f3dafad36695a4f8d20c5d41299e474e8c57
Signed-off-by: Manish Regmi <manish.regmi@intel.com>
---
 arch/x86/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 2488271..4337de2 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -106,7 +106,6 @@ config X86
 	select BUILDTIME_EXTABLE_SORT
 	select GENERIC_CMOS_UPDATE
 	select HAVE_ARCH_SOFT_DIRTY
-	select CLOCKSOURCE_WATCHDOG
 	select GENERIC_CLOCKEVENTS
 	select ARCH_CLOCKSOURCE_DATA if X86_64
 	select GENERIC_CLOCKEVENTS_BROADCAST if X86_64 || (X86_32 && X86_LOCAL_APIC)
-- 
1.9.1

